### PR TITLE
feat: add blog for observability v2.4 release updates

### PR DESCRIPTION
**Pull request by stijnmoreels**
Adds blog post for the Arcus Observability v2.4 release.

Relates #189

### CODE REVIEW

File: community/blog/2022-02 Service correlation preparation and .NET 6 support in Arcus Observability v2.4/README.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+# Service correlation preparation and .NET 6 support in Arcus Observability v2.4
+Some time has passed since our last release of Arcus Observability. Recently, we finished the v2.4 release which has tons of new features. Let's take a look!
+
+We are preparing for a major feature called [service-to-service correlation](https://docs.microsoft.com/en-us/azure/azure-monitor/app/app-map?tabs=net). This feature will allow us to correlate back-and-forth between different services, linking your entire application. This is a great way to get an solid overview of your project, and pin-point possible problems like bottlenecks.
```
Comment (pim-simons) [inline community/blog/2022-02 Service correlation preparation and .NET 6 support in Arcus Observability v2.4/README.md:4]:
```suggestion
We are preparing for a major feature called [service-to-service correlation](https://docs.microsoft.com/en-us/azure/azure-monitor/app/app-map?tabs=net). This feature will allow us to correlate back-and-forth between different services, linking your entire application. This is a great way to get a solid overview of your project and pin-point possible problems like bottlenecks.
```

File: community/blog/2022-02 Service correlation preparation and .NET 6 support in Arcus Observability v2.4/README.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+Because this feature encompasses several different systems and type of services, we are preparing each related Arcus repository. Starting with [Arcus Observability](https://github.com/arcus-azure/arcus.observability).
+
+## Tracking Azure Service Bus requests
+Before v2.4, we could only track HTTP requests. Which meant that we only could track incoming messages in an Web API context. For our service-to-service correlation feature, this was not enough as many different services can 'receive' a request. Azure Service Bus, for example.
```
Comment (pim-simons) [inline community/blog/2022-02 Service correlation preparation and .NET 6 support in Arcus Observability v2.4/README.md:8]:
```suggestion
Before v2.4, we could only track HTTP requests. Which meant that we could only track incoming messages in an Web API context. For our service-to-service correlation feature, this was not enough as many different services can 'receive' a request. Azure Service Bus, for example.
```

**Review by pim-simons — APPROVED** [review]
Great stuff 👍🏻

**Comment by stijnmoreels** [comment]
Thx @pim-simons for your detailed eye 😉 .

**Review by fgheysels — APPROVED** [review]

**Comment by stijnmoreels** [comment]
> LGTM !

Thx!
 
> Just wonder why we only support adding a dependencyId for KeyVault dependencies ?

Currently, we only support Azure Key Vault dependencies. But this will grow, of course. See https://github.com/arcus-azure/arcus.observability/issues/243#issuecomment-989640966 for the current state of this support.

**Pull request merged**